stdlib/public/TensorFlow/Execution.swift

```diff
@@ -20,24 +20,24 @@
 // function.  We need to inline if a callee contains tensor ops, not only if
 // it takes and returns a TensorFlow value.
 @_transparent
+@available(*, deprecated)
```

Comment:
IIRC, `deprecated` won't give you an error. `unavailable` will.

Reply:
This can be updated to unavailable once the tests are updated.